templates/layout/page_skeleton.html.twig

```diff
@@ -30,7 +30,10 @@
  # ---------------------------------------------------------------------
  #}
 
-{% do call(['Html', 'header'], {
+{% set interface = call("Session::getCurrentInterface") %}
```

Comment:
1. There is a Twig function for it.
2. We should avoid to use the `call` function in new features. It has been made to ease the transition to Twig but its usages have to be replaced by some cleaner code later.
```suggestion
{% set interface = get_current_interface() %}
```